mods/ts/rules/nod-support.yaml

```diff
@@ -41,6 +41,116 @@ NAGATE_B:
 	Tooltip:
 		Name: Nod Gate
 
+NAPOST:
+	Inherits: ^Building
+	Buildable:
+		Queue: Defense
+		BuildPaletteOrder: 1001
+		Prerequisites: naapwr, ~structures.nod
+		Description: Stops infantry and blocks enemy fire.\nCan NOT be crushed by tanks.
+	SoundOnDamageTransition:
+		DamagedSound:
+		DestroyedSound:
+	Valued:
+		Cost: 200
+	CustomSellValue:
+		Value: 0
```

Comment:
IIRC the posts actually sold for 100 credits in the original, only the segments didn't.